js/src/wp-seo-babel-polyfill.js

```diff
@@ -1 +1,5 @@
-require( "babel-polyfill" );
+// The babel polyfill sets the _babelPolyfill to true. So only load it ourselves if the variable is undefined or false.
+if ( typeof window._babelPolyfill !== "undefined" || ! window._babelPolyfill ) {
```

Comment:
`typeof window._babelPolyfill !== "undefined"` should be `typeof window._babelPolyfill === "undefined"`